cmd/fdo_client/client.go

```diff
@@ -386,6 +396,31 @@ func di() (err error) { //nolint:gocyclo
 	return err
 }
 
+func getDeviceInfo() (string, error) {
```

Comment:
Since this is Linux-specific, we should put it in a file with the appropriate build tags.

Reply:
will do, good point (I should  shift my mind to this being linux specific :stuck_out_tongue: )

Reply:
fixed